Update interfaces/web-bluetooth.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/52b74b6/whatwg/idl/web-bluetooth.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/466372583

foolip commented:
@lukebjerring no custom status check on this one. Is that a bug?

lukebjerring commented:
Digging into it now.

lukebjerring commented:
I don't think it's this PR itself causing it, but there's a few strange differences https://staging.wpt.fyi/results/bluetooth/idl/idlharness.tentative.https.window.html?product=chrome%5Bmaster%5D&product=chrome%4066693c4cb8&diff

autofoolip committed:
Update interfaces/web-bluetooth.idl

Source: https://github.com/tidoust/reffy-reports/blob/52b74b6/whatwg/idl/web-bluetooth.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/466372583
